dbt_common/events/helpers.py

```diff
@@ -9,6 +9,6 @@ def datetime_to_json_string(dt: datetime) -> str:
 
 # preformatted time stamp
 def get_json_string_utcnow() -> str:
-    ts = datetime.utcnow()
+    ts = datetime.now(timezone.utc)
```

Comment:
Hey @cfernhout this would return a time-zone aware object whereas `datetime.utcnow()` returns a naive datetime object. I think we will need to do something like this.

```suggestion
    ts = datetime.now(timezone.utc).replace(tzinfo=None)
```